github-filter/src/index.ts

```diff
@@ -138,7 +138,13 @@ async function sendWebhook(id: string, token: string, data: Data) {
   });
 
   // Pass on data to Discord as usual
-  return await fetch(template, new_request);
+  const response = await fetch(template, new_request);
+  if (!response.ok) {
+    return new Response(`Upstream error from Discord.`, {
```

Comment:
Can we include any further info from discord's response?

Reply:
What we could do is replace the static text with `response.json()` and purely just proxy the response back, that might be more useful